app/views/users/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>This is for the show</h1>
\ No newline at end of file
+<h2>Details of a selected user</h2>
+
+<span><img src="<%= @user.photo %>" alt="User Photo"></span>
+<span><%= @user.name %></span>
+
+<p><%= @user.bio %></p>
+
+<% @user.return_recent_posts.each_with_index do |post, index|%>
+<p>Post #<%= index+1 %></p>
+<p>
+  <%= post.text%>
+</p>
+ <span>Likes : <%= post.likes_counter %></span>
+  <span>Comments : <%= post.comments_counter %></span>
+<% end %>
+<%= link_to "See all posts", user_posts_path(@user.id)%>
```

Comment:
| Your Design | Given Design |
| --- | --- |
| ![image](https://github.com/Cabraham1/Blog/assets/93039359/c2b54370-d179-43be-9fc1-bc8e9e7b093f) | ![image](https://github.com/Cabraham1/Blog/assets/93039359/8ba7e254-90d8-40af-8f59-763faa16fdfa) |


- [ ] Great work on adding the image 👍 but I guess it would be great if you follow the wireframe as suggested by the previous reviewer that your image should be of the same size as the details are which you can see in the wireframe 😊

- [ ] As suggested by the previous reviewer make all your pages look like the given wireframe 👍 so, It would be nice if you organize your user details because if you compare your design with the given design then you would find a lot of things such s padding, margins, spaces, etc which are not similar 🤔 Kindly make these changes to make your project follow the given design 👍